docusaurus/docs/form/upload/upload.md

```diff
@@ -54,31 +56,31 @@ Callback to be executed when file is uploaded. The callback is provided the `Upl
 
 #### The `onFileUpload` callback takes precedence over the combination of `fileDeliveryMetadata`, `deliveryChannel`, and `deliverFileSubmit`. If all four properties are defined, only `onFileUpload` will be used.
 
-#### `onFileRemove?: (files: Array<File>, fileId: String) => void`
+#### `onFileRemove?: (files: File[], fileId: String) => void`
 
 Callback called when file is removed. The callback is provided two arguments. 1. the updated files and 2. the id of the file that was removed
 
-### `fileDeliveryMetadata?: object | (upload: UploadCore) => object`
+#### `fileDeliveryMetadata?: object | (upload: UploadCore) => object`
 
 The metadata properties that have been configured for the delivery channel you are trying to reach with `avFilesDeliveryApi`.
 
-#### Use this prop together with `deliveryChannel` and `deliverFileOnSubmit` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
+Use this prop together with `deliveryChannel` and `deliverFileOnSubmit` to have uploads automatically sent to the file delivery API with the proper configuration, without the need to define a custom `onFileUpload` function.
```

Comment:
I think we should add an h5 level here called something like `##### Usage with File Delivery API`, because we still want to call out this sentence and the following paragraph